client/src/main/resources/css/light.css

```
@@ -1,20 +1,89 @@
-.button{
-	-fx-background-color: orangered;
+* {
+	-fx-text-fill: black;
 }
 
```

Isn't the default text color black?
I'd suppose that this is unnecessary.
```suggestion
```

---

Could be, but I would like to keep it as it is good for clarity.

---

I do not think that it is making the code much clearer. Who'll ever take a look back at these lines? The only times I'm looking at them is when I'm changing them. And that is an operation that is rather impactful on the overall speed of Envoy as it will be applied to **every single** node that uses these CSS files. That accumulates over time.
So, in my opinion this makes the code not clearer AND is wasting time. Hence it is unnecessary.
I wonder what @CyB3RC0nN0R thinks about that.

---

Explicitly setting the color to black implies that it is something different by default, which can be misleading. In this case I agree with @delvh and urge you to remove the statement for the sake of clarity.